gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/forgot_password.html

```diff
@@ -85,35 +85,6 @@
     </div>
 </div>
 
-<!--[if lt IE 10]>
-<script th:src="@{assets/js/placeholder.js}"></script>
-<![endif]-->
-<script th:src="@{assets/material/material.min.js}"></script>
 <script th:src="@{assets/js/jquery-3.5.1.min.js}"></script>
-<script th:inline="javascript" th:attr="nonce=${script_inline_nonce}">
- $(document).ready(function (){
-    $(".mdl-textfield__input").focus(function (){
-        if( !this.value ){
-            $(this).prop('required', true);
-            $(this).parent().addClass('is-invalid');
-        }
-    });
-
-    $(".mdl-button").filter(":button").click(function (event){
```

Comment:
This section is used to enable the Google ReCaptcha on the page if the application settings requires this feature, so the code of this click event should be adapt to make ti works IMHO. (FYI, reCaptcha is define in the following forms login, registration, forgot_password and id_first_login)

Reply:
I did that at first, no problem.